engines/director/detection_tables.h

```diff
@@ -7054,7 +7054,7 @@ static const DirectorGameDescription gameDescriptions[] = {
 	WINGAME1("shramerica", "Print Activities", "PRINTPAK.EXE", "t:bb0fd2312d141d8cc05f77eb01c54885", 917721, 501),
 	// From SHR Exploration Station, Win version is misspelled on disc
 	MACDEMO1("shramerica", "Demo", "America Rock Demo", "d7c07df0d9f0176877a3c7c095ee0143", 705417, 500),
-	WINDEMO1("shramerica", "Demo", "amercia.exe", "t:c4906b1e01232db6ae2f1e8e365d92a4", 917695, 501),
```

Comment:
The code comment in the line above seems to be correct. On the disc for "Schoolhouse Rock: Exploration Station" there is a misspelled file ``amercia.exe`` with a matching file size.

Reply:
oh, i missed that comment, reverting

thank you